.env

```diff
@@ -22,7 +22,7 @@ STACKS_CORE_RPC_PORT=20443
 # STACKS_FAUCET_NODE_PORT=<port number>
 
 ## configure the chainID/networkID; testnet: 0x80000000, mainnet: 0x00000001
-STACKS_CHAIN_ID=0x00000001
+STACKS_CHAIN_ID=0x80000000
```

Comment:
I'm wondering if it might make more sense to set this env var in the relevant npm scripts? Otherwise, this change will fix the chain/network ID error when running in the dev/testnet environments, but introduce the error in mainnet environments. 

Alternatively, we can keep this change here, but ensure the `mainnet` variable is set in the dockerfiles?

Reply:
I added this as I figured it's a more sensible default, but strictly speaking, neither should really be defined. Can we remove it altogether?

Reply:
We could, as long as the dockerfiles specify them. But it would mean all current methods of starting the API (either in mainnet/testnet/mocknet), would immediately throw an error that this env var is missing. I think it makes more sense for the npm scripts that are most likely to be used to start in testnet/mocknet should ensure this is set.

Reply:
I think changing this is too disruptive without also changing a bunch of related docker images and probably some docs. Can you revert it?